etc/releasing.md

```diff
@@ -160,14 +194,27 @@ git reset --hard r1.2.3
 git push -f origin releases/stable
 ```
 
+## Branch if necessary
+
+If doing a new minor release `X.Y.0` (e.g. a `1.2.0` release), create branch
+`releases/vX.Y` (e.g `releases/v1.2`): `git checkout -b releases/v1.2 master`
+
+Push the new branch:
+
+```
+git push --set-upstream origin releases/v1.2
+```
+
+The new branch should be continuously tested on Evergreen. Update the "Display Name" and "Branch Name" of the [mongo-cxx-driver-latest-release Evergreen project](https://spruce.mongodb.com/project/mongo-cxx-driver-latest-release/settings/general) to refer to the new release branch.
+
 ## Generate and Publish Documentation
 
 Documentation generation must be run after the release tag has been made and
 pushed.
 
-- Checkout the master branch.
+- Create a new branch to contain documentation updates from master: `git checkout -b post-release-changes master`. This branch will be used to create a PR later.
```

Comment:
```suggestion
- Create and checkout a new branch to contain documentation updates from master: `git checkout -b post-release-changes master`. This branch will be used to create a PR later.
```
Emphasis the new branch should be checked out for the following steps (editing `etc`, `docs`, etc.).

On the other hand, if `post-release-changes` is _not_ the branch we should be on during the edits, add a bullet describing what branch we should checkout instead. Prior instructions suggest we should be on the `master` branch after this step:

```
- Create a new branch ...: `git branch post-release-changes master`. This branch will be used to create a PR later.
- Checkout the `master` branch.
- Edit ...
```

The explicit instruction to checkout the `master` branch should account for the various possible currently-checked-out branches that may result from prior instructions.

Note: `git checkout -b <branchname> <ref>` creates _and_ checks out a new branch `<branchname>` made on `<ref>`. To create but _not_ check out a new branch, `git branch <branchname> <ref>` should be used instead (or simply `git branch <branchname>` if we are already on `<ref>`).